lib/pbench/server/database/models/server_settings.py

```diff
@@ -269,7 +265,7 @@ def get(key: str, use_default: bool = True) -> "ServerSetting":
             if setting is None and use_default:
                 setting = ServerSetting(key=key, value=__class__._default(key))
         except SQLAlchemyError as e:
-            raise ServerSettingSqlError("finding", key, str(e)) from e
+            raise ServerSettingSqlError(e, operation="get", key=key) from e
```

Comment:
Does "get" read correctly in the exception message?  (We don't want it to end in "-ing", like it used to?)

Reply:
They weren't exactly full sentences before, and I generally went more towards reflecting the method name instead of verbing everything.